src/Parser.js

```diff
@@ -481,17 +493,38 @@ export default class Parser {
      * @param {"]" | "}"} breakOnTokenText - character to stop parsing the group on.
      * @return {?ParseNode}
      */
-    parseImplicitGroup(breakOnTokenText) {
+    parseImplicitGroup(breakOnTokenText?: "]" | "}" | "$") {
         const start = this.parseSymbol();
 
         if (start == null) {
             // If we didn't get anything we handle, fall back to parseFunction
             return this.parseFunction();
+        } else if (start.type === "arg") {
+            // Defer to parseGivenFunction if it's not a function we handle
+            return this.parseGivenFunction(start);
         }
 
         const func = start.result;
 
-        if (func === "\\left") {
+        if (func === "$") {
```

Comment:
I moved this up to the top so that we handle the case when `parseSymbol` returns a `Dollar` object first so that `flow` doesn't complain when it sees `oldFontFuncs[func]`.